src/loadsave.jl

```diff
@@ -115,20 +115,17 @@ savestreaming
 # if a bare filename or IO stream are given, query for the format and dispatch
 # to the formatted handlers below
 for fn in (:load, :loadstreaming, :save, :savestreaming, :metadata)
-    @eval $fn(s::Union{AbstractString,IO}, args...; options...) =
-        $fn(query(s), args...; options...)
+    @eval $fn(s, args...; options...) = $fn(query(s), args...; options...)
```

Comment:
Should we also make the variable name here more verbose?
```suggestion
    @eval $fn(stream, args...; options...) = $fn(query(stream), args...; options...)
```

Reply:
It doesn't have to `stream` but that's what `s` brought to mind

Reply:
Since it could be an a stream or filename I'm inclined to say `file`?

Reply:
That seems reasonable to me